### ISSUE TITLE
Using the web editor seems to changes something in null ?

### ISSUE THREAD

**Issue by calumk** [pocketbase/pocketbase#4068]
**v20.0.1**

Lets say i have a record, generated via API, that contains a field that is null.
lets call this field `demo_field`

Querying and filtering by null fields works as expected.
```
  let resp = await pb.collection('demo').getFullList({
    filter: 'demo_field=null'
  });
```

(This will return an array, containing the records with the null field

If i use the web editor, to edit one of the other fields in the same record (not the null field), and click save...

Now the field X is still null, shows up as null in the editor, and if i dont filter, the property is still returned as null.... but not longer filters as null...

```
  let resp = await pb.collection('demo').getFullList({
    filter: 'demo_field=null'
  });
```

This same response will return an array, containing the records with the null fields, but not including the manually edited record

Dont think im going crazy, but somethig here is a odd.

**Comment by ganigeorgiev** [comment]
What is the type of the `demo_field`?
Is this in a view collection?
I'll have a more detailed look at it tomorrow, but in the meantime could you provide some minimal reproducible steps?

**Comment by ganigeorgiev** [comment]
Oh, I think I know why this may happen with `json` fields that have `null` values because the serialized json `null` value is different than the SQL `NULL`.

I'll investigate it further tomorrow and will submit a query normalization fix with v0.20.2 release.

**Comment by calumk** [comment]
This would make sense, it is a JSON field

I remember you recently added the Json validation, so might be something related to that

Sorry I don't have a reproduction at the moment

For now I have solved this by just requesting all records and filtering in js later

**Comment by ganigeorgiev** [comment]
Should be fixed with [v0.20.3 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.20.3).
